src/twisted/conch/test/test_transport.py

```diff
@@ -3013,6 +3015,34 @@ def test_makeMAC(self):
                 f"Failed HMAC test vector; key={key!r} data={data!r}",
             )
 
+    def test_deprecation(self):
+        """
+        Test deprecation warning for legacy ciphers `CAST5`, `Blowfish`.
+
+        A deprecation warning is emitted when directly importing the
+        SSHCiphers class.
+        """
```

Comment:
Maybe the descript can be brief.

I don't think that SSHCiphers is relevant.
From the code, it looks like the warning is raised even if SSHCiphers is not used.

```suggestion
    def test_deprecationCAST5Blowfish(self):
        """
        A deprecation warning is raised when importing the L{twisted.conch.transport} module.
        """
```